### PR TITLE
Mes-4246 Modify the mobile app to use the new enums

### PR DESCRIPTION
**Pull request by appleabs**
## Description
Mes-4246 Modified the mobile app to use the latest schema, updated the mobile app to use the new enums.

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "@ngrx/effects": "^5.2.0",
     "@ngrx/store": "^5.2.0",
     "angular2-signaturepad": "^2.8.0",
+    "ci": "^1.0.0",
```
Comment (ammarhaiderbjss) [inline package.json]: What's this?

File: src/providers/fault-summary/fault-summary.ts [hunk 2]
```diff
@@ -210,7 +211,9 @@ export class FaultSummaryProvider {
   private getManoeuvreFaultsCatB (manoeuvres: CatBUniqueTypes.Manoeuvres, faultType: CompetencyOutcome)
   : FaultSummary[] {
     const faultsEncountered: FaultSummary[] = [];
-    forOwn(manoeuvres, (manoeuvre, type: ManoeuvreTypes) => {
+
+    // TODO: Replace any with Manoeuvres and change the transform function
```
Comment (ammarhaiderbjss) [inline src/providers/fault-summary/fault-summary.ts:215]: What's stopping us doing this now?

**Review by ammarhaiderbjss — CHANGES_REQUESTED** [review]
Great stuff 👍 
Not sure about some of the new dependencies added in `package-lock.json`.

**Review by ammarhaiderbjss — APPROVED** [review]
Brill, thank you

**Review by MarkPatt89 — APPROVED** [review]

**Pull request merged**